.github/workflows/go.yml

```diff
@@ -0,0 +1,28 @@
+name: lint-and-test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  lint-test-and-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
```

Comment:
Use Go 1.19 since 1.20 has been released and 1.18 is officially unsupported.

Reply:
fixed thanks